lms/djangoapps/teams/static/teams/js/views/topic_teams.js

```diff
@@ -68,7 +65,9 @@
 
                 showCreateTeamForm: function (event) {
                     event.preventDefault();
-                    Backbone.history.navigate('topics/' + this.model.id + '/create-team', {trigger: true});
+                    Backbone.history.navigate(
+                        'topics/' + this.model.id + '/create-team', {trigger: true}
```

Comment:
Nit: I'd put the second parameter on a new line as it is a little hidden on the same line.